### PR TITLE
Validation enums

### PR DESCRIPTION
**Pull request by Ikumi123**
I did some validations, and want to try pulling, pls let me know if there is anything wrong 

image

image

image

### CODE REVIEW

File: app/models/anime.rb [hunk 1]
```diff
@@ -1,2 +1,4 @@
 class Anime < ApplicationRecord
+  validates title:, presence: true
+  validates title:, uniqueness: true
```
Comment (ayanofujita) [inline app/models/anime.rb:3]:
```suggestion
  validates :title, presence: true
  validates :title, uniqueness: true
```

File: app/models/review.rb [hunk 2]
```diff
@@ -2,4 +2,6 @@ class Review < ApplicationRecord
   belongs_to :reviewer, class_name: "User"
   belongs_to :reviewee, class_name: "User"
   belongs_to :order
+  validates :bio, length: { minumum: 10 }
+  validates :bio, length: { maximum: 1000 }
```
Comment (ayanofujita) [inline app/models/review.rb:6]:
```suggestion
  validates :bio, length: { in: 10..1000 }
```

File: app/models/user.rb [hunk 3]
```diff
@@ -3,6 +3,10 @@ class User < ApplicationRecord
   # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
   devise :database_authenticatable, :registerable,
           :recoverable, :rememberable, :validatable
+  validates :username, :first_name, :lastname, :address, :bio, presence: true
+  validates :username, :first_name, :lastname, :bio, uniqueness: true
+  validates :bio, length: { minimum: 10 }
+  validates :bio, length: { maximum: 500 }
```
Comment (ayanofujita) [inline app/models/user.rb:9]:
```suggestion
  validates :bio, length: { in: 10..500 }
```

**Review by PuvaSan — APPROVED** [review]
Looks good! Thanks Ikumi

**Pull request merged**